src/components/svg/Fav.svg

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
As a recommendation, always [use a tool](https://jakearchibald.github.io/svgomg/) to clean up your generated SVGs.